modules/light-clients/08-wasm/types/store_test.go

```diff
@@ -373,21 +384,15 @@ func (suite *TypesTestSuite) TestNewMigrateClientWrappedStore() {
 	}
 }
 
-// GetSubjectAndSubstituteStore returns a subject and substitute store for testing.
+// GetSubjectAndSubstituteStore returns two dummy KVStores for testing the migrate client wrapping store.
 func (suite *TypesTestSuite) GetSubjectAndSubstituteStore() (storetypes.KVStore, storetypes.KVStore) {
-	suite.SetupWasmWithMockVM()
-
-	endpointA := wasmtesting.NewWasmEndpoint(suite.chainA)
-	err := endpointA.CreateClient()
-	suite.Require().NoError(err)
-
-	subjectClientStore := suite.chainA.App.GetIBCKeeper().ClientKeeper.ClientStore(suite.chainA.GetContext(), endpointA.ClientID)
+	suite.SetupTest()
 
-	substituteEndpoint := wasmtesting.NewWasmEndpoint(suite.chainA)
-	err = substituteEndpoint.CreateClient()
-	suite.Require().NoError(err)
+	ctx := suite.chainA.GetContext()
+	storeKey := GetSimApp(suite.chainA).GetKey(types.StoreKey)
 
-	substituteClientStore := suite.chainA.App.GetIBCKeeper().ClientKeeper.ClientStore(suite.chainA.GetContext(), substituteEndpoint.ClientID)
+	subjectClientStore := prefix.NewStore(ctx.KVStore(storeKey), []byte("client1"))
+	substituteClientStore := prefix.NewStore(ctx.KVStore(storeKey), []byte("client2"))
```

Comment:
Not to leave any banana peels or misunderstandings lying around it might be good to adapt this to:
- Take `storeKey` as the ibc store key, as this is what client stores are built from
- Make the byte strings some proper client ID. E.g. `08-wasm-0`, `08-wasm-1`

Should be able to use `clienttypes.FormatClientIdentifier(types.Wasm, 0)` in here I think.